bin/dbServer.py

```diff
@@ -31,12 +31,33 @@
 from flask import Flask, request
 import json
 import logging as log
+import os
 import sys
 
+import ConfigParser
+import sqlalchemy
```

Comment:
I guess this is a new dependency, table file needs to include dependency on anaconda (which has sqlalchemy)